src/pages/workspace/expensifyCard/WorkspaceExpensifyCardDetailsPage.tsx

```diff
@@ -111,6 +120,7 @@ function WorkspaceExpensifyCardDetailsPage({route}: WorkspaceExpensifyCardDetail
                                 title={formattedAvailableSpendAmount}
                                 interactive={false}
                                 titleStyle={styles.newKansasLarge}
+                                containerStyle={isOffline ? styles.buttonOpacityDisabled : null}
```

Comment:
<img width="1512" alt="Screenshot 2024-08-15 at 16 15 49" src="https://github.com/user-attachments/assets/5120a163-043c-4e3f-8b8b-b4642b573704">

@mountiny @VickyStash We always greyed out the remaining limit field when offline (even though there are no pending action), right?

Reply:
@DylanDylann Correct because we cant know if the data is correct / up to date